content/en/docs/concepts/traffic-management/index.md

```diff
@@ -529,7 +529,7 @@ traffic for services running outside of the mesh, including the following tasks:
 -   Define [retry](#retries), [timeout](#timeouts), and
     [fault injection](#fault-injection) policies for external destinations.
 -   Add a service running in a Virtual Machine (VM) to the mesh to
-    [expand your mesh](/docs/examples/mesh-expansion/single-network/#running-services-on-a-mesh-expansion-machine).
+    [expand your mesh]/docs/tasks/virtual-machines/single-network/#running-services-on-a-mesh-expansion-machine).
```

Comment:
Is the # anchor correct? It has a substring "mesh-expansion" in it.

Reply:
Also replace [expand your mesh] with [add vms]?